var/spack/repos/builtin/packages/xsdk-examples/package.py

```diff
@@ -103,5 +103,5 @@ def enabled(pkg):
         return args
 
     def check(self):
-        with working_dir(self.build_directory):
+        with working_dir(self.builder.build_directory):
```

Comment:
@alalazo,  FYI:

This looks like a general Spack issue:

After merging the latest `develop` branch in this PR, `self.build_directory` was no longer working with error message
```console
'XsdkExamples' object has no attribute 'build_directory'
```
I had to replace it with `self.builder.build_directory` to get it to work.